changelog/fragments/kb-800fdeec6e5c.yaml

```diff
@@ -0,0 +1,72 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Golang/Ansible/Helm/HybridHelm language-based operators (go/v3, ansible/v1, helm/v1, hybrid.helm/v1-alpha): applying restrictive SCC for all containers scaffolded by the tool (`k8s versions < 1.19 will no longer work with`).
+      ([More info](https://github.com/kubernetes-sigs/kubebuilder/pull/2700)). Important: You might be affect from Kubernetes 1.25 if you do not ensure that the containers
+      used in your project are restrictive or are defined if the property labels `pod-security.kubernetes.io` on clusters ([More info](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)). Therefore, you suggest
+      you begin to configure your projects accordingly. ([More info](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/))
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+    migration:
```

Comment:
Since this is a user-facing change in scaffolding, don't we need migration guide?

Reply:
is it about the pod restrictive? if yes, we added to see the migration steps.